client/src/views/menu/MenuPage.vue

```diff
@@ -0,0 +1,69 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <sidebar-menu
+      v-if="userInfo"
+      :user-info="userInfo"
+    />
+    <tab-menu
+      v-if="userInfo"
+      v-show="menuMode === MenuMode.Tabs"
+      :user-info="userInfo"
+    />
+  </ion-page>
+</template>
+
+<script setup lang="ts">
+import { ClientInfo, getClientInfo as parsecGetClientInfo } from '@/parsec';
+import { IonPage } from '@ionic/vue';
+import { Ref, onMounted, onUnmounted, ref } from 'vue';
+import TabMenu from '@/views/menu/TabMenu.vue';
+import SidebarMenu from '@/views/menu/SidebarMenu.vue';
+
+enum MenuMode {
+  Sidebar = 'sidebar',
+  Tabs = 'tabs',
+}
+
+const userInfo: Ref<ClientInfo | null> = ref(null);
+const menuMode = ref<MenuMode>(MenuMode.Sidebar);
+
+async function updateWindowWidth(): Promise<void> {
+  if (window.innerWidth <= 768) {
```

Comment:
`setToastOffset` should be called here. `0` for Tabs, and a fixed value for Sidebar.